### ISSUE TITLE
Notification Popup in bottom right corner

### ISSUE THREAD

**Issue by metas-mk** [metasfresh/metasfresh-webui#61]
Notification Popup in bottom right corner
- additionally to Notfication menu
- have a popup for each notification with small metasfresh logo and name in header
- show for x seconds and then fade away
- user has the same information and link in that notification as in Notification menu

**Comment by damianprzygodzki** [comment]
Should it exchange current one in upper right corner?

**Comment by metas-mk** [comment]
Yes. Should only have one place where notification infos popup.

**Comment by teosarca** [comment]
@metas-mk is this still valid?

**Comment by metas-mk** [comment]
Yes and no. The idea I had at that time is that it should look like a browser notification with metasfresh logo/ icon. We agreed that it died not make sense at that time and would be better to leave at the top. Currently, I still feel that the notification popups (the blue ones and the red error ones are really disturbing and don't look like if they fit properly from a design perspective) and we should think about what/ how to change.

**Comment by teosarca** [comment]
Ok, but in that case i would really consider one of the followings (might be also complementary):
* http://caniuse.com/#feat=push-api
* http://caniuse.com/#feat=notifications

The biggest benefit would be that are and will be better integrated with the native platform like Android.
